src/qililab/typings/enums.py

```diff
@@ -334,6 +338,7 @@ class ResultName(str, Enum):
 
     QBLOX = "qblox"
     VECTOR_NETWORK_ANALYZER = "vector_network_analyzer"
+    QM = "quantum_machines"
```

Comment:
I would prefer to give the full name descriptions. Eg. `QUANTUM_MACHINES = "quantum_machines"`. In the same sense that `VECTOR_NETWORK_ANALYZER` was not named just `VNA`. It is more understandable for everyone and easier to search in VS Code.

Reply:
agreed, changed.